File: tasks/python-feature-002/verify.py
"""Verifier for python-feature-002."""
from text_utils import slugify

cases = {
    "Hello, World!": "hello-world",
    "  Already--Spaced  ": "already-spaced",
    "Python_3.11": "python-3-11",
    "***": "",
}
for raw, expected in cases.items():
    actual = slugify(raw)
    if actual != expected:
        raise SystemExit(f"slugify({raw!r}) returned {actual!r}, expected {expected!r}")

File: tasks/javascript-feature-012/verify.mjs
import { slugify } from "./slug.js";

const cases = new Map([
  ["Hello, World!", "hello-world"],
  ["  Already--Spaced  ", "already-spaced"],
  ["JS_2026", "js-2026"],
  ["***", ""],
]);
for (const [raw, expected] of cases) {
  if (slugify(raw) !== expected) throw new Error(`slugify failed for ${raw}`);
}
